src/TableV8/index.tsx

```diff
@@ -17,21 +17,38 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import { ColumnDef, getCoreRowModel, useReactTable } from '@tanstack/react-table';
-import { StyledTable, StyledTableBody, StyledTableCell, StyledTableRow } from './styled';
+import { ColumnDef, flexRender, getCoreRowModel, useReactTable } from '@tanstack/react-table';
+import {
+  StyledTableContainer,
+  StyledTableHeader,
+  StyledTable,
+  StyledTableBody,
+  StyledTableCell,
+  StyledTableHead,
+  StyledTableRow,
+  StyledResizableTableHeaderContent,
+} from './styled';
 
 interface ReactTableProps<TData> {
   className?: string;
   columns: ColumnDef<TData>[];
   data: TData[];
-  isStriped?: boolean;
+  withHeaders?: boolean;
+  withRowBorder?: boolean;
+  withRowHighlight?: boolean;
+  withSideBorders?: boolean;
+  withStripes?: boolean;
 }
 
 export const TableV8 = <TData extends object>({
-  className,
-  columns,
-  data,
-  isStriped = false,
+  className = '',
+  columns = [],
+  data = [],
+  withHeaders = false,
+  withSideBorders = false,
+  withRowHighlight = false,
+  withStripes = false,
+  withRowBorder = false,
```

Comment:
these customisations seem useful, as a start, but then with all the possible combinations a user may want, the props may grow exponentially. I think there could be a use here for enabling explicit CSS customisation.

Reply:
very true... i'm copying the old tables for now

Reply:
figured as much, therefor approved regardless